core/state/database.go

```diff
@@ -237,6 +241,9 @@ func (db *cachingDB) StartVerkleTransition(originalRoot, translatedRoot common.H
 	if pragueTime != nil {
 		chainConfig.PragueTime = pragueTime
 	}
+	if err := db.disk.Put(keyVerkleTransitionEnded, []byte{0}); err != nil {
+		panic(err) // This is fine since this branch is only used for replay
+	}
```

Comment:
The minimum amount of state that we need to persist is if the transition ended, so we do that in this new db key.